src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,89 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * TaskList object stores the List of Tasks
+ */
+public class TaskList {
+    private List<Task> list;
+
+    public TaskList(List<Task> list) {
+        this.list = list;
+    }
+
+    public TaskList() {
+        this.list = new ArrayList<>();
+    }
+
+    public List<Task> getList() {
+        return this.list;
+    }
+
+    /**
+     * Iterates through the list of Tasks stored
+     * @return A String of List of Tasks stored
+     */
+    public String iterateToDo() {
```

Comment:
I think ierateAndPrintTodo might be a name that better captures the purpose of this method.

Reply:
I agree